db/schema.sql

```diff
@@ -136,7 +136,8 @@ CREATE TABLE report_aggregations(
     ord                 BIGINT NOT NULL,                    -- a value used to specify the ordering of client reports in the aggregation job
     state               REPORT_AGGREGATION_STATE NOT NULL,  -- the current state of this report aggregation
     prep_state          BYTEA,                              -- the current preparation state (opaque VDAF message, only if in state WAITING)
-    prep_msg            BYTEA,                              -- the next preparation message to be sent to the helper (opaque VDAF message, only if in state WAITING if this aggregator is the leader)
+    prep_msg            BYTEA,                              -- for the leader, the next preparation message to be sent to the helper (opaque VDAF message)
+                                                            -- for the helper, the next preparation share to be sent to the leader (opaque VDAF message)
```

Comment:
Could we spell out in which states this should be non-NULL? (is it when in the `WAITING` state for both leader and helper cases?)